python/kwiver/sprokit/tests/sprokit/pipeline/test-datum.py

```diff
@@ -124,19 +125,13 @@ def test_error_():
 
 
 if __name__ == '__main__':
-    import os
     import sys
 
-    if not len(sys.argv) == 4:
-        test_error("Expected three arguments")
+    if not len(sys.argv) == 2:
```

Comment:
Minor nitpick: `not len(sys.argv) == 2` -> `len(sys.argv) != 2`.

Reply:
Easy to make this change via:

```bash
SEARCH="not len(sys.argv) =="
REPLACE="len(sys.argv) !="
find . -type f -iname '*.py' -exec sed -i "s|${SEARCH}|${REPLACE}|g" {} + 
```

Reply:
That worked, thanks